examples/java-agent/build.gradle

```diff
@@ -24,8 +24,8 @@ repositories {
 }
 
 dependencies {
-  compile "com.sparkjava:spark-core:2.9.1"
-  compile "org.slf4j:slf4j-simple:1.7.28"
+  implementation 'com.sparkjava:spark-core:2.9.1'
```

Comment:
Just to double check, does this also works for older gradle versions (before 7) ?

Reply:
Yeah, should work with sufficiently ancient Gradle versions.